index.d.ts

```diff
@@ -451,17 +491,17 @@ export type ExecaSyncError<StdoutStderrType extends StdoutStderrAll = string> =
 	/**
 	Error message when the child process failed to run. In addition to the underlying error message, it also contains some information related to why the child process errored.
 
-	The child process stderr then stdout are appended to the end, separated with newlines and not interleaved.
+	The child process `stderr` then `stdout` are appended to the end, separated with newlines and not interleaved.
 	*/
 	message: string;
 
 	/**
-	This is the same as the `message` property except it does not include the child process stdout/stderr.
+	This is the same as the `message` property except it does not include the child process `stdout`/`stderr`.
 	*/
 	shortMessage: string;
 
 	/**
-	Original error message. This is the same as the `message` property except it includes neither the child process stdout/stderr nor some additional information added by Execa.
+	Original error message. This is the same as the `message` property except it includes neither the child process `stdout`/`stderr` nor some additional information added by Execa.
```

Comment:
Understanding this diff is a formatting update, the sentence reads a little confusing to me (particularly from  `except it includes neither [...]`).

Light suggestion, but perhaps this is more clear:
> This is the same as the `message` property excluding the child process `stdout`/`stderr` and additional information added by Execa.

Reply:
Much better! Done.